kinto/authorization.py

```diff
@@ -25,7 +25,9 @@
 # Ex: bucket:read is granted by both bucket:write and bucket:read
 PERMISSIONS_INHERITANCE_TREE = {
     '': {  # Granted via settings only.
-        'bucket:create': {},
+        'bucket:create': {
+            '': ['bucket:create', 'create']
+        },
```

Comment:
Are you sure this is necessary? I would say that the code in `from_settings()` is enough.